src/audio/kpb.c

```diff
@@ -834,7 +833,7 @@ static uint64_t kpb_draining_task(void *arg)
 			comp_update_buffer_produce(sink, size_to_copy);
 	}
 
-	time_end =  platform_timer_get(platform_timer);
+	timer =  platform_timer_get(platform_timer) - timer;
```

Comment:
@lgirdwood We need config in CI that build something else that default config. F.e. debug or no-trace builds. Most developers while doing PRs just checks default config, just like CI. Maintaining traceless build that can fail on things like that will be hard.

Reply:
@jajanusz 
Can we change the config with some scripts? I prefer to do that with some scripts inside this repo so that we can make sure we can cover all config once it comes out.

Reply:
@xiulipan You can change single configs like in scripts/xtensa-build-all.sh